draft-ietf-anima-brski-prm.md

```diff
@@ -139,25 +140,24 @@ The approach defined here is agnostic to the enrollment protocol that connects t
 
 # Introduction
 
-BRSKI as defined in {{RFC8995}} specifies a solution for secure zero-touch (automated) bootstrapping of devices (pledges) in a (customer site) domain.
```

Comment:
"customer site domain" was not fully cleaned up as discussed.
Section on nomadic connectivity refers to "installation location", otherwise "site" not used anymore.